src/Microsoft.OData.ModelBuilder/Microsoft.OData.ModelBuilder.xml

```diff
@@ -2777,6 +2782,11 @@
             The EntitySetPathExpression that entities are returned from.
             </summary>
         </member>
+        <member name="P:Microsoft.OData.ModelBuilder.OperationConfiguration.VocabularyTermConfigurations">
+            <summary>
+            Vocabulary builders to annotate this <see cref="T:Microsoft.OData.ModelBuilder.OperationConfiguration"/>
```

Comment:
@g2mula A huge PR has just been merged to odata.net repo to warnings, some emanating from using cref tags with prefix https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1200?view=vs-2019

Reply:
This XML is auto generated at build time from the C# source files. The source files do not contain prefixes.